li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -33,6 +33,7 @@
   public static final String MESSAGE_ASSEMBLER_EXPIRATION_OFFSET_GAP_CONFIG = "message.assembler.expiration.offset.gap";
   public static final String MAX_TRACKED_MESSAGES_PER_PARTITION_CONFIG = "max.tracked.messages.per.partition";
   public static final String EXCEPTION_ON_MESSAGE_DROPPED_CONFIG = "exception.on.message.dropped";
+  public static final String EXCEPTION_ON_INVALID_OFFSET_CONFIG = "exception.on.invalid.offset";
```

Comment:
rename to : 
exception.on.invalid.offset.reset?
an exception is thrown anyway if offset reset strategy is none.